integration-cli/docker_cli_login_test.go

```diff
@@ -14,8 +15,7 @@ func (s *DockerSuite) TestLoginWithoutTTY(c *check.C) {
 	cmd.Stdin = bytes.NewBufferString("buffer test string \n")
 
 	// run the command and block until it's done
-	if err := cmd.Run(); err == nil {
-		c.Fatal("Expected non nil err when loginning in & TTY not available")
-	}
+	err := cmd.Run()
+	c.Assert(err, checker.NotNil, check.Commentf("Expected non nil err when loginning in & TTY not available"))
```

Comment:
I mean that both are fine by me :
```go
c.Assert(err, checker.NotNil, check.Commentf("Expected non nil err when loginning in & TTY not available"))
// or
c.Assert(err, checker.NotNil) // Expected non nil err when loginning in & TTY not available"
```

I'm used to do the `check.Commentf`` so I tend to prefer using it :cat:.